core/src/functionalTest/groovy/com/novoda/gradle/release/AndroidDifferentGradleVersions.groovy

```diff
@@ -0,0 +1,134 @@
+package com.novoda.gradle.release
+
+import org.assertj.core.api.Assertions
+import org.gradle.testkit.runner.GradleRunner
+import org.gradle.testkit.runner.TaskOutcome
+import org.junit.*
+import org.junit.rules.TemporaryFolder
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4.class)
+class AndroidDifferentGradleVersions {
+
+    @Rule
+    public TemporaryFolder temporaryFolder = new TemporaryFolder()
+
+    private GradleRunner runner
+
+    @Before
+    void setUp() throws Exception {
+        runner = GradleRunner.create()
+                .withProjectDir(temporaryFolder.root)
+                .withArguments("build", "bintrayUpload", "-PbintrayKey=key", "-PbintrayUser=user")
+                .withPluginClasspath()
+
+        File manifestFile = new File(temporaryFolder.root, "/src/main/AndroidManifest.xml")
+        manifestFile.getParentFile().mkdirs()
+        manifestFile.createNewFile()
+        manifestFile.write('<manifest package="com.novoda.test"/>')
+
+        def buildFile = temporaryFolder.newFile("build.gradle")
+        buildFile.write(
+                """            
+            buildscript {
+                repositories {
+                    jcenter()
+                    google()
+                }
+                dependencies {
+                    classpath 'com.android.tools.build:gradle:3.0.0'
+                    classpath 'com.novoda:bintray-release:0.8.0'
+                }
+            }
+            
+            apply plugin: "com.android.library"
+            apply plugin: "com.novoda.bintray-release"
+            
+            android {
+                compileSdkVersion 26
+                buildToolsVersion "26.0.2"
+
+                defaultConfig {
+                    minSdkVersion 16
+                    versionCode 1
+                    versionName "0.0.1"
+                }    
+                
+                lintOptions {
```

Comment:
what about disabling lint completely? this will speed up testing time. It could be disabled here or we can add `-x lint` to the test build as an argument to skip the task.